ynr/apps/cached_counts/tests/test_cached_counts.py

```diff
@@ -155,27 +155,27 @@ def test_attention_needed_page(self):
             [
                 (
                     "<td>2015 General Election</td>",
-                    '<td><a href="/election/2015/post/65913/camberwell-and-peckham">Camberwell and Peckham</a></td>',
+                    '<td><a href="/elections/2015.65913/">Camberwell and Peckham</a></td>',
                     "<td>0</td>",
                 ),
                 (
                     "<td>Maidstone local election</td>",
-                    '<td><a href="/election/local.maidstone.2016-05-05/post/DIW:E05005004/shepway-south-ward">Shepway South Ward</a></td>',
+                    '<td><a href="/elections/local.maidstone.DIW:E05005004.2016-05-05/">Shepway South Ward</a></td>',
```

Comment:
I know this is all test data, so its somewhat contrived, but is there any value in trying to have the tests use slighty more realistic values here. If anything that looks even vaguely like `2015.65913` ever exists as a real ballot id, something went wrong. e.g:

* `2015.65913` would be `parl.camberwell-and-peckham.2015-05-07` in real life
* `local.maidstone.DIW:E05005004.2016-05-05` would be `local.maidstone.shepway-south-ward.2016-05-05`

etc

if this is loads of effort, we can ignore it.

Reply:
This one is probably something to think about for future as opposed to something that should block this PR - just mentioning it while its staring me in the face.

Reply:
Yes, this is actually a proxy for updating how the tests work. A lot of them were written _before_ `PostExtraElection` was a model in the project, meaning that we set up a `Post` and an `Election` and test just using them. If I were writing them now (especially after this PR) I would want to think of the _ballot_ as the main unit of test fixtures, adding elections and posts as needed.

This looks like changing [`UK2015ExamplesMixin`](https://github.com/DemocracyClub/yournextrepresentative/blob/84f3e5f355cb4efa7cbdf9291647b021db68a023/ynr/apps/candidates/tests/uk_examples.py#L50), but over 45 files use that mixin, so I think it would be a fairly large chunk of work.

I'll make an issue for it.